qemu/tests/cdrom.py

```diff
@@ -802,12 +857,16 @@ def test(self):
 
             if self.is_src:  # Starts in source
                 self.cdrom_new = create_iso_image(params, "new")
+                serial_num = generate_serial_num()
+                cdrom = params["cdroms"]
+                params["drive_serial_%s" % cdrom] = serial_num
```

Comment:
since there is no driver_serial_cdrom in cfg file. should we add start_vm =no in cfg and start VM here ?